From 2e3744b9f9e64fbed18513027c0d000cbc8d046b Mon Sep 17 00:00:00 2001
From: Nick Schermer <nick@xfce.org>
Date: Sun, 08 May 2011 21:00:07 +0000
Subject: Remove mime-types from the exo helper desktop files (bug #7257).

The desktop files are only used to advertise the preferred application
framework in menus and launchers. It should not show up when opening
applications, since the user can choose the native application for
that.
---
diff --git a/exo-open/exo-file-manager.desktop.in b/exo-open/exo-file-manager.desktop.in
index 8d5fa13..ca4aedf 100644
--- a/exo-open/exo-file-manager.desktop.in
+++ b/exo-open/exo-file-manager.desktop.in
@@ -7,6 +7,5 @@ StartupNotify=true
 Terminal=false
 Categories=Utility;X-XFCE;X-Xfce-Toplevel;
 OnlyShowIn=XFCE;
-MimeType=x-scheme-handler/file;x-scheme-handler/trash;
 _Name=File Manager
 _Comment=Browse the file system
diff --git a/exo-open/exo-mail-reader.desktop.in b/exo-open/exo-mail-reader.desktop.in
index be971c6..541de0d 100644
--- a/exo-open/exo-mail-reader.desktop.in
+++ b/exo-open/exo-mail-reader.desktop.in
@@ -7,6 +7,5 @@ StartupNotify=false
 Terminal=false
 Categories=Network;X-XFCE;X-Xfce-Toplevel;
 OnlyShowIn=XFCE;
-MimeType=x-scheme-handler/mailto;
 _Name=Mail Reader
 _Comment=Read your email
diff --git a/exo-open/exo-web-browser.desktop.in b/exo-open/exo-web-browser.desktop.in
index c323753..65870e5 100644
--- a/exo-open/exo-web-browser.desktop.in
+++ b/exo-open/exo-web-browser.desktop.in
@@ -7,7 +7,6 @@ StartupNotify=false
 Terminal=false
 Categories=Network;X-XFCE;X-Xfce-Toplevel;
 OnlyShowIn=XFCE;
-MimeType=x-scheme-handler/http;x-scheme-handler/https;
 _Name=Web Browser
 _Comment=Browse the web
 
--
cgit 
